src/authentication/promiscuous.hpp

```diff
@@ -1,15 +1,17 @@
-#include "cocaine/api/auth.hpp"
+#include "cocaine/api/authentication.hpp"
 
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
 #include <boost/lexical_cast.hpp>
 
+#include "cocaine/auth/uid.hpp"
+
 namespace cocaine {
-namespace auth {
+namespace authentication {
 
-class promiscuous_t : public api::auth_t {
+class promiscuous_t : public api::authentication_t {
```

Comment:
"promiscuous_t" is not suitale here - it just allways returns anonymous credentials